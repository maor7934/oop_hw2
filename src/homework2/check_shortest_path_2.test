CreateNode n1 5
CreateNode n2 1
CreateNode n3 3
CreateNode n4 2
CreateNode n5 10
CreateNode n6 3
CreateNode n7 3
CreateNode n8 5
CreateNode n9 21
CreateNode n10 21
CreateNode n11 20
CreateGraph A
AddNode A n1
AddNode A n2
AddNode A n3
AddNode A n4
AddNode A n5
AddNode A n6
AddNode A n7
AddNode A n8
AddNode A n9
AddNode A n10
AddNode A n11
AddEdge A n1 n4
AddEdge A n1 n5
AddEdge A n1 n6
AddEdge A n2 n4
AddEdge A n2 n5
AddEdge A n2 n6
AddEdge A n3 n4
AddEdge A n3 n5
AddEdge A n3 n6
AddEdge A n4 n7
AddEdge A n6 n8
AddEdge A n5 n9
AddEdge A n5 n10
AddEdge A n5 n11
AddEdge A n7 n9
AddEdge A n7 n10
AddEdge A n7 n11
AddEdge A n8 n9
AddEdge A n8 n10
AddEdge A n8 n11
FindPath A n1 n2 n3 -> n9 n10 n11
FindPath A n1 -> n10
AddEdge A n3 n9
FindPath A n3 -> n9
