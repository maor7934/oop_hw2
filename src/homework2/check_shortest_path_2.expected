created node n1 with cost 5
created node n2 with cost 1
created node n3 with cost 3
created node n4 with cost 2
created node n5 with cost 10
created node n6 with cost 3
created node n7 with cost 3
created node n8 with cost 5
created node n9 with cost 21
created node n10 with cost 21
created node n11 with cost 20
created graph A
added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to A
added node n5 to A
added node n6 to A
added node n7 to A
added node n8 to A
added node n9 to A
added node n10 to A
added node n11 to A
added edge from n1 to n4 in A
added edge from n1 to n5 in A
added edge from n1 to n6 in A
added edge from n2 to n4 in A
added edge from n2 to n5 in A
added edge from n2 to n6 in A
added edge from n3 to n4 in A
added edge from n3 to n5 in A
added edge from n3 to n6 in A
added edge from n4 to n7 in A
added edge from n6 to n8 in A
added edge from n5 to n9 in A
added edge from n5 to n10 in A
added edge from n5 to n11 in A
added edge from n7 to n9 in A
added edge from n7 to n10 in A
added edge from n7 to n11 in A
added edge from n8 to n9 in A
added edge from n8 to n10 in A
added edge from n8 to n11 in A
shortest path in A: n2 n4 n7 n11
shortest path in A: n1 n4 n7 n10
added edge from n3 to n9 in A
shortest path in A: n3 n9
